packages/vite/src/client/overlay.ts

```diff
@@ -150,7 +164,7 @@ kbd {
 </div>
 `
 
-const fileRE = /(?:[a-zA-Z]:\\|\/).*?:\d+:\d+/g
+const fileRE = /(?:[a-z]:\\|\/).*?:\d+:\d+/gi
```

Comment:
Oddly enough, this seems to be changed automatically, but it looks like it is working the same, do I need to change it back? 🤔

Reply:
I've disabled it here: <https://github.com/vitejs/vite/blob/0435ba733d4dfedfbc9b4c9400c342b2d3e70ab6/.eslintrc.cjs#L111-L112>. Perhaps it was formatted accidentallly during merge? Could try reverting it that change and see if it reformats again.